src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -114,7 +114,10 @@ function MoneyRequestReportPreviewContent({
     isInvoice,
     shouldShowBorder = false,
     onPress,
+    chatReportMetadata,
 }: MoneyRequestReportPreviewContentProps) {
+    const shouldShowLoading = !chatReportMetadata?.hasOnceLoadedReportActions && transactions.length === 0;
+    const shouldShowLoadingDeferredValue = useDeferredValue(shouldShowLoading);
```

Comment:
```suggestion
    const shouldShowLoadingDeferred = useDeferredValue(shouldShowLoading);
```

IMO the `value` part is redundant. The phrase `deferred` itself can be used as a noun and I think it's ok. Just like `shouldShowPromise` would make sense, you can say `shouldShowDeferred`

Reply:
@Kicu @SzymczakJ Could you give a comment to explain why we need to use useDeferredValue here?